src/restartscript.cpp

```diff
@@ -197,34 +197,7 @@ static const char* cmdlineArgHandler =
   "fi\n\n"
 ;
 
-static const char* singleHostProcessing =
-  "ckpt_files=\"\"\n"
-  "if [ ! -z \"$DMTCP_RESTART_DIR\" ]; then\n"
-  "  for tmp in $given_ckpt_files; do\n"
-  "    ckpt_files=\"$DMTCP_RESTART_DIR/$(basename $tmp) $ckpt_files\"\n"
-  "  done\n"
-  "else\n"
-  "  ckpt_files=$given_ckpt_files\n"
-  "fi\n\n"
-
-  "coordinator_info=\"--coord-host $coord_host --coord-port $coord_port\"\n"
-
-  "tmpdir=\n"
-  "if [ ! -z \"$DMTCP_TMPDIR\" ]; then\n"
-  "  tmpdir=\"--tmpdir $DMTCP_TMPDIR\"\n"
-  "fi\n\n"
-
-  "ckpt_dir=\n"
-  "if [ ! -z \"$DMTCP_CKPT_DIR\" ]; then\n"
-  "  ckpt_dir=\"--ckptdir $DMTCP_CKPT_DIR\"\n"
-  "fi\n\n"
-
-  "exec $dmt_rstr_cmd $coordinator_info $ckpt_dir \\\n"
-  "  $maybejoin --interval \"$checkpoint_interval\" $tmpdir $noStrictChecking \\\n"
-  "  $ckpt_files\n"
-;
-
-static const char* multiHostProcessing =
+static const char* rstrCmdProcessing =
```

Comment:
Let's change the name from `rstrCmdProcessing` to `rstrtCmdProcessing`.  (This distinguishes "restore" from "restart".)